polyA/load_alignments.py

```diff
@@ -180,6 +180,20 @@ def load_alignments(
             seqs.clear()
 
 
+def load_alignment_tool(file: TextIO) -> str:
+    """
+    Return alignment tool used.
+    """
+    for line in file:
+        if _parse_preamble_line(line):
+            continue
+        if line.strip().upper().startswith("# ALIGNMENT TOOL"):
+            # get alignment tool
+            return line.split()[-1]
+        return ""
```

Comment:
Why is this here? It causes the loop to only ever run once, right? If that's the behavior you want, the more standard way to do it would be:

```python
try:
  line = next(file)
except StopIteration:
  return ""

# Do stuff with line
```